error.js

```diff
@@ -0,0 +1,106 @@
+export function ErrorCheck(code, data = "") {
```

Comment:
일반적으로 함수 네이밍은 파스칼케이스를 사용하지않고, 동사로 시작합니다 :)

```suggestion
export function checkError(code, data = "") {
```

아래 아티클 참고해보시면 도움이 되실것같아요!
[참고](https://velog.io/@cada/%EC%9E%90%EB%B0%94%EC%8A%A4%ED%81%AC%EB%A6%BD%ED%8A%B8-%EC%8A%A4%ED%83%80%EC%9D%BC-%EA%B0%80%EC%9D%B4%EB%93%9C-%EB%84%A4%EC%9D%B4%EB%B0%8D-%EC%BB%A8%EB%B2%A4%EC%85%98-%ED%8E%B8)